src/command_modules/azure-cli-webapp/azure/cli/command_modules/webapp/tests/test_webapp_commands.py

```diff
@@ -145,6 +145,11 @@ def body(self):
         self.assertTrue('s2' not in result)
         self.assertTrue('s3' in result)
 
+        self.cmd('appservice web config hostname list -g {} --webapp {}'.format(self.resource_group, self.webapp_name), checks=[
+            JMESPathCheck('length(@)', 1),
+            JMESPathCheck('[0].name', '{}/{}.azurewebsites.net'.format(self.webapp_name, self.webapp_name))
```

Comment:
Instead of doing `'{}/{}...'.format(self.webapp_name, self.webapp_name)`, you can do `'{0}/{0}...'.format(self.webapp_name)`
see https://docs.python.org/2/library/string.html#format-examples